engine/src/main/java/io/zeebe/engine/processor/workflow/handlers/multiinstance/MultiInstanceBodyCompletedHandler.java

```diff
@@ -69,35 +73,24 @@ protected boolean handleMultiInstanceBody(
     return true;
   }
 
-  private void updateOutputCollection(
+  protected void updateOutputCollection(
```

Comment:
Why did you change the method visibility?

```suggestion
  private void updateOutputCollection(
```

Reply:
oops 😄